packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -1,34 +1,52 @@
 <template>
     <template if:true={hasParentsOrValues}>
         <lightning-card title={label}>
-            <ul class="slds-has-dividers_around-space slds-m-bottom_medium">
-                <template if:true={hasParents}>
-                    <lightning-button variant="base" label={labels.clear} title={labels.clear} icon-name="utility:clear"
-                                      class="slds-m-left_medium" onclick={reset}></lightning-button>
-                </template>
-                <template for:each={parents} for:item="v">
-                    <li class="slds-m-horizontal--medium slds-m-vertical--x-small slds-grid" key={v.value}>
-                        <span class="nowrap slds-text-title_bold">{v.value}</span>
+            <lightning-button-icon
+                class="slds-var-p-bottom_small facet__collapse"
+                slot="actions"
+                onclick={toggleFacetVisibility}
+                icon-name={collapseIconName}
+                alternative-text="Collapse/show facet"
```

Comment:
This string should be localized.

Reply:
I actually added a label for this already in the recent-results PR.